src/main/java/org/folio/circulation/domain/policy/LoanPolicy.java

```diff
@@ -467,6 +467,7 @@ private String getProfileId(JsonObject loansPolicy) {
     return loansPolicy.getString("profileId");
   }
 
+  @Override
   public String getName() {
     return representation.getString("name");
   }
```

Comment:
Why is this a virtual overridden property? Can the value be provided as part of the constructor (and passed to the base class) and hence remove the need for the JSON representation here?

Reply:
Because I don't want to change loan Policy as it is out of scope of this story. The loan policy is used in a lot of places in the code and it is a risk to break something.

Reply:
I understand the concern that it might break something.

I don't think I follow, why is this change out of the scope of this work?

If the change isn't made during this work, when should it be made?

Reply:
Changed